argus/src/main/java/com/moldedbits/argus/provider/BaseProvider.java

```diff
@@ -70,7 +68,7 @@ public void onClick(View v) {
      *
      * @return Login button id
      */
-    protected int getActionButtonId() {
+    private int getActionButtonId() {
```

Comment:
this is not supposed to be private as child providers can have different action button ids